code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -12,9 +12,22 @@
 	equip_sound = 'sound/items/equip/toolbelt_equip.ogg'
 	/// Do we have overlays for items held inside the belt?
 	var/use_item_overlays = FALSE
+	/// Bypasses the "belt in bag" prevention if TRUE
+	var/storable = FALSE
+	/// Ignores update_weight() if TRUE
+	var/large = FALSE
+
```

Comment:
```suggestion

/obj/item/storage/belt/Initialize(mapload)
	. = ..()
	update_weight()

```
You can do this instead of adding `update_weight()` to the end of all these `populate_contents()` proc overrides

Reply:
Does Init(mapload) only affect its initial spawn rounstart? The issue I had was with using the Game Panel to spawn stuff in and it not updating weight until after it "updated" by putting new tools in/out.

Reply:
aaand done, thank ya.